src/current/v23.1/sql-audit-logging.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: SQL Audit Logging
+title: Table-based SQL Audit Logging
 summary: Use the EXPERIMENTAL_AUDIT setting to turn SQL audit logging on or off for a table.
 toc: true
 docs_area: manage
```

Comment:
Suggest adding "table-based" to the first sentence below (applies to both versions)

Reply:
Also suggest adding Role-based SQL Audit Logging to the See More links section

Reply:
Updated